__tests__/marshalls.repo.test.js

```diff
@@ -0,0 +1,123 @@
+const RepoMarshall = require('../lib/marshalls/repo.marshall')
+const urlExists = require('url-exists')
```

Comment:
tbh I was thinking at first that we'll just check if there's a repository URL string that exists at all and not necessarily check if it's alive or not. It's a nice added check but I guess someone could just add the google homepage as a URL and this will pass this check too, no?

if we do end up actually pinging a URL then maybe let's search for a different package since this one is 2 years old? I would even argue that we can just do a simple HTTP call ourselves and check the returned status code. Since we already have axios this should be like 2 lines of code to do.